OTAnalytics/plugin_parser/track_export.py

```diff
@@ -0,0 +1,58 @@
+from pandas import DataFrame
+
+from OTAnalytics.application.use_cases.track_export import (
+    ExportTracks,
+    TrackExportSpecification,
+)
+from OTAnalytics.domain import track
+from OTAnalytics.domain.track_repository import TrackRepository
+from OTAnalytics.plugin_datastore.track_store import (
+    FilteredPandasTrackDataset,
+    PandasTrackDataset,
+)
+
+
+class CsvTrackExport(ExportTracks):
```

Comment:
Do we need to write a test case for this class?

Reply:
Done.